common/source_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// package common defines types common to parsing and other diagnostics.
+package common
+
+import (
+	"testing"
+)
+
+const (
+	UnexpectedValue   = "Expected '%v', got '%v'"
+	UnexpectedSnippet = "Expected snippet '%s', got '%s'"
+	SnippetNotFound   = "Expected snippet at line %d, but not found"
+	SnippetFound      = "Found snippet at line %d, where none was expected"
```

Comment:
It looks like these four constants are not used outside this file.  If so, suggest making them non-exported (initial lowercase).